release-checklist.md

```diff
@@ -0,0 +1,21 @@
+# Release checklist
+- [ ] `git flow release start v19.3.0` (`v19.3.0-beta.1` if beta)
+- [ ] Check that all new changes are reflected in the changelog
+- [ ] Bump package.json version to `v19.3.0` (`v19.3.0-beta.1` if beta)
```

Comment:
I don't see the `v` in the `package.json` so:

- [ ] Bump the `version` in the `package.json` version to `VERSION_NUMBER` leaving off the `v` designation. 
    For example, `19.3.0` or 19.3.0-beta.1` are both valid `VERSION_NUMBER` values.